WallPaperDev/WallPaperDev/Views/Controllers/MilestonePromptVC.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  MilestonePromptVC.swift
+//  WallPaperDev
+//
+//  Created by Jackson Ho on 4/16/20.
+//  Copyright © 2020 Stephen Ouyang. All rights reserved.
+//
+
+import UIKit
+
+protocol passMilestoneData: class {
+  func passMilestone(_ description: String)
+}
+
+final class MilestonePromptVC: UIViewController {
+  
+  private var formView: MilestoneFormView = MilestoneFormView()
+  private var keyboardHeight: CGFloat = 0
+  weak var delegate: passMilestoneData?
+  
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    view.backgroundColor = .black
+    setupUI()
+    setupKeyboardNotifications()
+  }
+  
+  private func setupKeyboardNotifications() {
```

Comment:
If we end up needing this in a few places we could move this logic to a BaseViewController

Reply:
I agree since one of the common interactions in the app is filling forms. I will research other ways to handle this so it can be configurable for other VC.

Reply:
Nice!